include/vctr/TypeTraitsAndConcepts/ContainerAndExpressionConcepts.h

```diff
@@ -358,12 +358,6 @@ concept inputIteratorToConstructValuesOfType = std::input_iterator<T> && std::co
 template <class T, class ElementType>
 concept contiguousIteratorWithValueTypeSameAs = std::contiguous_iterator<T> && std::same_as<ElementType, std::iter_value_t<T>>;
 
-/** Constrains the type to be suitable for initializing a single element Vctr, that is, it is not of any other type
-    for which special single argument constructors exist, so no ambiguity can arise.
- */
-template <class T>
-concept suitableSingleElementInitializer = ! anyVctrOrExpression<T> && ! triviallyCopyableWithDataAndSize<T> && ! suitableInitializerFunction<T>;
```

Comment:
I'm actually not sure why I decided to chose this rather specific blacklist approach back then instead of the more straightforward approach that I have chosen now ;)